src/renderers/createFusionRenderer.spec.luau

```diff
@@ -1,7 +1,6 @@
 local Fusion = require("@pkg/Fusion")
 local JestGlobals = require("@pkg/JestGlobals")
 
-local createFusionRenderer = require("./createFusionRenderer")
```

Comment:
I'm not sure if I like the indirection where the renderer being tested is now implicitly picked up in tests. For the `render` API I think that's great, but it would be nice to have an escape hatch